templates/docs/patterns/lists/index.md

```diff
@@ -90,11 +90,9 @@ inline list items.
 View example of the middot list pattern
 </a></div>
 
-The utility class `.is-dark` can also be applied to a middot list:
+The utility class `.is-dark` can also be applied to a middot list.
```

Comment:
I don't know if that was called like that in other PRs, `is-dark` is not an utility class. Utilities are class names starting with `u-`. This is a modifier class (or in this specific case a theme class, that can be applied anythere… which actually makes it functionally similar to utility…).

Anyway, in this case, this documentation bit is really not needed anymore. Let's remove it. We will be likely removing it from other components as well sooner or later.